README.md

```diff
@@ -2070,66 +2075,56 @@ $ ssh-add -E md5 -l
 
-On modern distributions, such as Fedora 30, there is typically no need to also set `RemoteForward` in `~/.ssh/config` as detailed in the next chapter, because the right thing actually happens automatically.
-
+### Use S.gpg-agent.ssh
 
-### Steps for older distributions
+First you need to go through [Remote Machines (GPG Agent Forwarding)](#remote-machines-gpg-agent-forwarding), know the conditions for gpg-agent forwarding and know the location of `S.gpg-agent.ssh` on both the local and the remote.
 
-On the local machine, run:
+You may use the command:
 
 ```console
-$ gpgconf --list-dirs agent-extra-socket
+$ gpgconf --list-dirs agent-ssh-socket
 ```
 
-This should return a path to agent-extra-socket - `/run/user/1000/gnupg/S.gpg-agent.extra` - though on older Linux distros (and macOS) it may be `/home/<user>/.gnupg/S/gpg-agent.extra`
-
-Find the agent socket on the **remote** machine:
-
-```console
-$ gpgconf --list-dirs agent-socket
-```
-
-This should return a path such as `/run/user/1000/gnupg/S.gpg-agent`
-
-Finally, enable agent forwarding for a given machine by adding the following to the local machine's ssh config file `~/.ssh/config` (your agent sockets may be different):
+Then in your `.ssh/config` add one sentence for that remote
 
 ```
 Host
   Hostname remote-host.tld
-  ForwardAgent yes
-  RemoteForward /run/user/1000/gnupg/S.gpg-agent /run/user/1000/gnupg/S.gpg-agent.extra
+  StreamLocalBindUnlink yes
+  RemoteForward /run/user/1000/gnupg/S.gpg-agent.ssh /run/user/1000/gnupg/S.gpg-agent.ssh
   # RemoteForward [remote socket] [local socket]
+  # Note that ForwardAgent is not wanted here!
 ```
 
-If you're still having problems, it may be necessary to edit `gpg-agent.conf` file on both the remote and local machines to add the following information:
+After successfully ssh into the remote, you should check that you have `/run/user/1000/gnupg/S.gpg-agent.ssh` lying there.
 
-```
-enable-ssh-support
-pinentry-program /usr/bin/pinentry-curses
-extra-socket /run/user/1000/gnupg/S.gpg-agent.extra
+The in the *remote* you can type in command line or configure in the shell rc file with
+
+```console
+export SSH_AUTH_SOCK="/run/user/$UID/gnupg/S.gpg-agent.ssh"
 ```
 
-See [Issue #85](https://github.com/drduh/YubiKey-Guide/issues/85) for more information and troubleshooting.
+After typing or sourcing your shell rc file, with `ssh-add -l` you should find your ssh public key now.
+
+**Note** In this process no gpg-agent in the remote is involved, hence `gpg-agent.conf` in the remote is of no use. Also pinentry is invoked locally.
+
+**Note** Agent forwarding may be chained through multiple hosts
```

Comment:
Can we have a short example of what say a 2-hop chain would look like?

Reply:
Added in the new commit